testspec.yml

```diff
@@ -9,6 +9,7 @@ phases:
       - start-dockerd
       - pip install -r src/requirements.txt
       - python src/send_status.py --status 2
+      - apt-get update && apt-get install unzip
```

Comment:
Lets install the `unzip` package on the codebuild image itself and also `apt-get update` if required